src/connection/Connection.ts

```diff
@@ -407,6 +415,13 @@ export class Connection {
         return relationMetadata.junctionEntityMetadata;
     }
 
+    /**
+     * creates an Entity Manager with the help of the EntityManagerFactory
```

Comment:
First latter is capital + missing dot at the end of comment. Why method is public? Also, why it accepts a connection? Its not logically correct for a single connection to have a "createEntityManager" method which can accept other connections.